src/Component/Configuration/GridConfigurationSortingHandler.php

```diff
@@ -26,6 +26,10 @@ public function handle(array $gridConfiguration): array
         foreach ($gridConfiguration['sorting'] as $sorting => $order) {
             Assert::keyExists($gridConfiguration['fields'] ?? [], $sorting);
 
+            if (isset($gridConfiguration['fields'][$sorting]['sortable'])) {
+                continue;
+            }
+
             $gridConfiguration['fields'][$sorting]['sortable'] = true;
         }
```

Comment:
I'm wondering if using `isset` here is the right solution.

Looking at the tests, I had to update the line for the author because its sortable value was originally being changed from `false` to `true`. However, with these changes, that no longer happens.

Maybe the value should be set to `true` if it's either not set or currently `false`.

Reply:
maybe should we throw an error that indicates the field is not sortable.